1_space_odyssey/index.html

```diff
@@ -10,7 +10,7 @@
 </head>
 <body class="fade-in">
     
-    <script src="./js/mannuel-9000/app.js"></script>
+    <script src="../1_space_odyssey/js/mannuel-9000/line-9000/app.js"></script>
```

Comment:
please create your folder so it make the path look like this.
`./js/line-9000/app.js`